packages/vuetify/src/components/VDataTable/VDataTable.ts

```diff
@@ -425,7 +428,10 @@ export default VDataIterator.extend({
 
       return this.$createElement(this.isMobile ? MobileRow : Row, {
         key: getObjectValueByPath(item, this.itemKey),
-        class: classes,
+        class: {
+          ...classes,
+          selected: data.isSelected,
```

Comment:
Shouldn't we use BEM here in other places (expanded, expanded__row)?

Reply:
mmm, I guess you're right